lib/ssoe/configuration.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/base'
+
+module SSOe
+  class Configuration < Common::Client::Configuration::SOAP
+    def self.ssl_cert_path
+      IdentitySettings.mvi.client_cert_path
+    end
+
+    def self.ssl_key_path
+      IdentitySettings.mvi.client_key_path
+    end
+
+    def ssl_options
+      if ssl_cert && ssl_key
+        {
+          client_cert: ssl_cert,
+          client_key: ssl_key
+        }
+      end
+    end
+
+    def base_path
+      if Settings.vsp_environment == 'staging'
+        'https://sqa.services.eauth.va.gov:9303/psim_webservice/IdMSSOeWebService' # staging
+      else
+        'https://int.services.eauth.va.gov:9303/psim_webservice/dev/IdMSSOeWebService' # dev
+      end
+    end
```

Comment:
Should we use `IdentitySettings.mvi.url` here or are we wanting to bypass the forward proxy?

Reply:
Bypassing fwd proxy